src/routes/event.js

```diff
@@ -0,0 +1,13 @@
+const { mockEvents } = require('../../database/events_schema.js');
+
+module.exports = (req, res) => {
+  const id = req.params.id;
```

Comment:
i think its only going to make harder if its for testing im going to push all the tests that i did for the routes soon, also for the readability it doesn't make any sense to split the files cause the one who is going to read the code is just going to have to jump between 2 files its bad DX ( developer experience :stuck_out_tongue_winking_eye: )